api-specificatie/openapi.yaml

```diff
@@ -706,7 +705,7 @@ components:
       schema:
         description: "Deze wordt gebruikt in de operation, bv. /woonplaatsen/0123. Waarde: 0001-9999."
         type: string
-        pattern: '[0-9]{3}[1-9]|[0-9]{2}[1-9][0-9]|[0-9][1-9][0-9]{2}|[1-9][0-9]{3}'
+        pattern: '^[0-9]{3}[1-9]|[0-9]{2}[1-9][0-9]|[0-9][1-9][0-9]{2}|[1-9][0-9]{3}$'
```

Comment:
Kan dit pattern niet veel eenvoudiger? Dus: '^[0-9]{4}$'
Het gaat erom dat de gebruiker weet wat het formaat is, in dit geval exact 4 cijfers. Dus bijvoorbeeld niet de woonplaatsnaam of code zonder voorloopnul. Voor het overige moet de gebruiker gewoon de juiste code invoeren en krijgt deze anders gewoon de juiste melding.

Reply:
@fsamwel Die opmerking heb je al eens eerder gemaakt (ik kan alleen het issue zo snel niet terugvinden). Toen kwamen we eropuit dat dit correct is.

Reply:
oké prima